Explorer/Assets/DCL/UI/ISpriteCache.cs

```diff
@@ -0,0 +1,15 @@
+using Cysharp.Threading.Tasks;
+using System.Threading;
+using UnityEngine;
+
+namespace DCL.UI
+{
+    public interface ISpriteCache
+    {
+        Sprite? GetCachedSprite(string imageUrl);
+        UniTask<Sprite?> GetSpriteAsync(string imageUrl, CancellationToken ct = default);
+        UniTask<Sprite?> GetSpriteAsync(string imageUrl, bool useKtx, CancellationToken ct = default);
+
+        void Clear();
```

Comment:
We are gonna need to implement a clear function for a specific image url. Talking with the back-end guys, they told me that, for the community edition endpoint, when we update the image, they will return the same url... so in this case if we don't remove manually that cached image, we're gonna show the non-updated image in the community.